src/App.tsx

```diff
@@ -1,61 +1,139 @@
+        user: addedUser,
+      },
+    ]));
+
+    setSelectedUser('');
+    setTitle('');
+  };
 
-export const App = () => {
   return (
     <div className="App">
-      <h1>Add todo form</h1>
-
-      <form action="/api/users" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>Choose a user</option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
-
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
-
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
-
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
-      </section>
+      <div className="wrapper">
+        <h1 className="title is-1">Add todo form</h1>
+        <form
+          action="/api/users"
+          method="POST"
+          onSubmit={handleFormSubmit}
+        >
+          <div className="field">
+            <label
+              className="title is-5"
+              htmlFor="#title"
+            >
+              Title:
+            </label>
+            <input
+              className="input is-small w-max"
+              id="title"
+              type="text"
+              data-cy="titleInput"
+              placeholder="Enter a title"
+              value={title}
+              onChange={handleTitleChange}
+            />
+            {isTitleError
+              && <span className="error">Please enter a title</span>}
+          </div>
+
+          <div className="field">
+            <label
+              className="title is-5"
+              htmlFor="#userSelect"
+            >
+              User:
+            </label>
+            <select
+              className="input is-small"
+              data-cy="userSelect"
+              id="userSelect"
+              value={selectedUser}
+              onChange={handleUserChange}
+            >
+              <option value="" disabled>Choose a user</option>
```

Comment:
```suggestion
              <option value="" disabled>Choose a user</option>
```

do not left empty value